app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardMobileActivity.java

```diff
@@ -647,12 +651,14 @@ public void updateTotalFilters(Integer totalFilters) {
     public void hideTabsAndDisableSwipe() {
         binding.tabLayout.setVisibility(View.GONE);
         binding.teiPager.setUserInputEnabled(false);
+        ViewCompat.setElevation(binding.toolbar, 0);
     }
 
     @Override
     public void showTabsAndEnableSwipe() {
         binding.tabLayout.setVisibility(View.VISIBLE);
         binding.teiPager.setUserInputEnabled(true);
+        ViewCompat.setElevation(binding.toolbar, elevation);
```

Comment:
Why elevation is always 0? Shouldn't it be set to the default value?

Reply:
Yes,  in OnCreate() I get the default value of the elevation of the toolbar. Here I set that value to the default. The elevation variable is never set to 0